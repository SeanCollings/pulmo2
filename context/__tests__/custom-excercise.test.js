import React, { useContext } from 'react';
import { create } from 'react-test-renderer';
import { cleanup } from 'react-native-testing-library';

import CustomExcerciseContextProvider, {
  CustomExcerciseContext,
} from '../custom-excercise-context';

const MockComponent = () => {
  const context = useContext(CustomExcerciseContext);
  expect(context).toMatchSnapshot();
  return null;
};

describe('custom-excercise-context', () => {
  afterEach(cleanup);

  test('unit test', () => {
    let context = create(
      <CustomExcerciseContextProvider>
        <MockComponent />
      </CustomExcerciseContextProvider>
    );

    expect(context).toMatchSnapshot();
  });
});
