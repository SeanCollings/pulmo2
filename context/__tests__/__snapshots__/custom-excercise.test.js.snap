// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom-excercise-context unit test 1`] = `
Object {
  "addCustomExcercise": [Function],
  "customExcercises": undefined,
  "deleteCustomExcercise": [Function],
  "updateCustomExcercise": [Function],
}
`;

exports[`custom-excercise-context unit test 2`] = `null`;
